R/estimateDiversity.R

```diff
@@ -284,7 +284,7 @@ setMethod("estimateDiversity", signature = c(x="TreeSummarizedExperiment"),
             ..., BPPARAM = SerialParam()){
         # input check
         # Check tree_name
-        if( !.is_non_empty_string(tree_name) ){
```

Comment:
General note: The package is already in use. That is why you should avoid making changes if possible. --> easier to debug and read the history --> also it does not look good if we keep on changing existing code

--> so changes are welcome but they should improve the code and be well-reasoned

Reply:
--> revert all changes regarding removal of space in if sentences